sktime/datasets/_readers_writers/tsf.py

```diff
@@ -0,0 +1,302 @@
+"""Functions for reading time series data from .tsf files."""
```

Comment:
typo?

Reply:
They are not. they are not duplicates. could you pinpoint which parts in these two files are duplicates?

`tsf` is the file format invented by the people that manage Monash Time Series Forecasting Repository while `ts` is the one that was invented in sktime.

Reply:
apologies, this was an inspection error.